### PR TITLE
3.3/cleanup

### PR DESCRIPTION
**Pull request by yakatz**
One error left in kohana-core, which is a mistake in the definition file.
There is a possibility that I broke the UTF-8 tests. Those files should probably be changed to hex encoded instead of the actual characters.

Now fixed the UTF-8 tests.

### CODE REVIEW

File: utf8/transliterate_to_ascii.php [hunk 1]
```diff
@@ -18,21 +18,21 @@ function _transliterate_to_ascii($str, $case = 0)
...
-				'ū' => 'u',  'č' => 'c',  'ö' => 'o',  'è' => 'e',  'ŷ' => 'y',  'ą' => 'a',  'ł' => 'l',
-				'ų' => 'u',  'ů' => 'u',  'ş' => 's',  'ğ' => 'g',  'ļ' => 'l',  'ƒ' => 'f',  'ž' => 'z',
-				'ẃ' => 'w',  'ḃ' => 'b',  'å' => 'a',  'ì' => 'i',  'ï' => 'i',  'ḋ' => 'd',  'ť' => 't',
-				'ŗ' => 'r',  'ä' => 'a',  'í' => 'i',  'ŕ' => 'r',  'ê' => 'e',  'ü' => 'u',  'ò' => 'o',
-				'ē' => 'e',  'ñ' => 'n',  'ń' => 'n',  'ĥ' => 'h',  'ĝ' => 'g',  'đ' => 'd',  'ĵ' => 'j',
-				'ÿ' => 'y',  'ũ' => 'u',  'ŭ' => 'u',  'ư' => 'u',  'ţ' => 't',  'ý' => 'y',  'ő' => 'o',
-				'â' => 'a',  'ľ' => 'l',  'ẅ' => 'w',  'ż' => 'z',  'ī' => 'i',  'ã' => 'a',  'ġ' => 'g',
-				'ṁ' => 'm',  'ō' => 'o',  'ĩ' => 'i',  'ù' => 'u',  'į' => 'i',  'ź' => 'z',  'á' => 'a',
-				'û' => 'u',  'þ' => 'th', 'ð' => 'dh', 'æ' => 'ae', 'µ' => 'u',  'ĕ' => 'e',  'ı' => 'i',
+				'Ã ' => 'a',  'Ã´' => 'o',  'Ä�' => 'd',  'á¸Ÿ' => 'f',  'Ã«' => 'e',  'Å¡' => 's',  'Æ¡' => 'o',
```
Comment (shadowhand) [inline utf8/transliterate_to_ascii.php]: Looks broke to me.

File: utf8/transliterate_to_ascii.php [hunk 2]
```diff
@@ -48,21 +48,21 @@ function _transliterate_to_ascii($str, $case = 0)
...
-				'Ū' => 'U',  'Č' => 'C',  'Ö' => 'O',  'È' => 'E',  'Ŷ' => 'Y',  'Ą' => 'A',  'Ł' => 'L',
-				'Ų' => 'U',  'Ů' => 'U',  'Ş' => 'S',  'Ğ' => 'G',  'Ļ' => 'L',  'Ƒ' => 'F',  'Ž' => 'Z',
-				'Ẃ' => 'W',  'Ḃ' => 'B',  'Å' => 'A',  'Ì' => 'I',  'Ï' => 'I',  'Ḋ' => 'D',  'Ť' => 'T',
-				'Ŗ' => 'R',  'Ä' => 'A',  'Í' => 'I',  'Ŕ' => 'R',  'Ê' => 'E',  'Ü' => 'U',  'Ò' => 'O',
-				'Ē' => 'E',  'Ñ' => 'N',  'Ń' => 'N',  'Ĥ' => 'H',  'Ĝ' => 'G',  'Đ' => 'D',  'Ĵ' => 'J',
-				'Ÿ' => 'Y',  'Ũ' => 'U',  'Ŭ' => 'U',  'Ư' => 'U',  'Ţ' => 'T',  'Ý' => 'Y',  'Ő' => 'O',
-				'Â' => 'A',  'Ľ' => 'L',  'Ẅ' => 'W',  'Ż' => 'Z',  'Ī' => 'I',  'Ã' => 'A',  'Ġ' => 'G',
-				'Ṁ' => 'M',  'Ō' => 'O',  'Ĩ' => 'I',  'Ù' => 'U',  'Į' => 'I',  'Ź' => 'Z',  'Á' => 'A',
-				'Û' => 'U',  'Þ' => 'Th', 'Ð' => 'Dh', 'Æ' => 'Ae', 'İ' => 'I',
+				'Ã€' => 'A',  'Ã”' => 'O',  'ÄŽ' => 'D',  'á¸ž' => 'F',  'Ã‹' => 'E',  'Å ' => 'S',  'Æ ' => 'O',
```
Comment (shadowhand) [inline utf8/transliterate_to_ascii.php]: And this too.

**Comment by kiall** [comment]
Yup - definitely broke!

**Comment by yakatz** [comment]
Will open a new pull request. Should be fixed now.

**Comment by kiall** [comment]
Wait - Why does that say I added some commits?? -_-

**Comment by yakatz** [comment]
@kiall Because my Git client is in beta and is broken. I forced the commit back to me as the author.

**Pull request closed without merging**